libs/sdk-ui-kit/src/Header/HeaderBadge.tsx

```diff
@@ -0,0 +1,23 @@
+// (C) 2021 GoodData Corporation
+import React from "react";
+import cx from "classnames";
+
+/**
+ * @internal
+ */
+export interface IHeaderBadgeProps {
+    className?: string;
+    color?: string;
+    backgroundColor?: string;
+}
+
+/**
+ * @internal
+ */
+export const HeaderBadge: React.FC<IHeaderBadgeProps> = ({ children, color, backgroundColor, className }) => {
+    return (
+        <div className={cx("gd-header-badge", className)} style={{ color, backgroundColor }}>
```

Comment:
Do we need className here?

Reply:
Hm, we probably don't I will remove it to keep the API more restrictive :)

Reply:
Removed